config/choria.go

```diff
@@ -81,6 +81,7 @@ type ChoriaPluginConfig struct {
 	NetworkSystemPassword              string        `confkey:"plugin.choria.network.system.password"`                                                             // Password used to access the Choria system account
 	NetworkSystemUsername              string        `confkey:"plugin.choria.network.system.user"`                                                                 // Username used to access the Choria system account
 	NetworkTLSTimeout                  int           `confkey:"plugin.choria.network.tls_timeout" default:"2"`                                                     // Time to allow for TLS connections to establish, increase on slow or very large networks
+	NetworkAuthTimeout                 int           `confkey:"plugin.choria.network.auth_timeout" default:"2"`                                                    // Time to allow for clients to authenticate, increase on slow or very large networks
```

Comment:
I think better to make this a duration and then when setting it call `.Seconds()` on it, gives people some more flexibility and its a bit more consistent with things like write deadline below

Reply:
Should maybe also error if this is over some silly value like 1 mnute